### PR TITLE
legend fill for peer review

### PR DESCRIPTION
**Pull request by jordansread**
@aappling-usgs can you look? 

This is unfinished because it doesn't add to the json. Pulled that out because I have to leave now and couldn't finish. 

The values don't seem right to me (values seem high relative to area). 

image

### CODE REVIEW

**Comment by eread-usgs** [comment]
👍

File: layout/js/map.js [hunk 1]
```diff
@@ -59,6 +73,7 @@ var animate_resize_map = function(data) {
       state.css(style);
     }
   });
+  document.getElementById('category-area-text').firstChild.data = legendVals[category].toLocaleString() + ' mgd water withdrawls ';// + category.replace(/_/g, ' '); removing because redundant w/ selector
```
Comment (aappling-usgs) [inline layout/js/map.js]: withdrawls -> withdrawals

**Review by aappling-usgs — COMMENTED** [review]
sounds like the proj string on the legend circles might do it. will wait on your edit. the state hovertext calculation makes sense to me.

**Comment by dblodgett-usgs** [comment]
👍  This looks good to me. You still looking for a JSON 'spert?

File: scripts/process/scaleStates.R [hunk 2]
```diff
@@ -33,8 +37,10 @@ process.scaleStates <- function(viz){
     thisCat <- mutate(thisCat, scaleFactor = newArea / area)
 
     finalScaleFactors <- bind_rows(finalScaleFactors, thisCat)
+    catvalues[[cat]] = legend.area * magicState$value / magicState$area
```
Comment (jiwalker-usgs) [inline scripts/process/scaleStates.R]: Can this be mutated into finalScaleFactors somehow (attributes)?  We don't have a good way of producing multiple outputs from a single function right now.

**Comment by dblodgett-usgs** [comment]
Nice. Looks good to me! Not sure you really needed another function for those categoryValues, but keeping the json stuff on it's own was a good idea. :shipit:

**Pull request merged**